rts/System/FileSystem/FileHandler.cpp

```diff
@@ -133,9 +133,8 @@ bool CFileHandler::TryReadFromRawFS(const string& fileName)
 	}
 
 	return true;
-#else
-	return false;
 #endif
+	return false;
```

Comment:
The non-TOOLS path will have two returns now, this may confuse static analysis (or produce warnings etc)